cmd/decrypt.go

```diff
@@ -63,12 +64,24 @@ func decryptRun(cmd *cobra.Command, args []string) {
 	var publickeyobj *crypto.Key
 	var publickeyring *crypto.KeyRing
 	publickeyobj, err = crypto.NewKeyFromArmored(string(decryptionpublickey))
+	if err != nil {
```

Comment:
Not necessary here, but I feel like this needs to become a RunE and we return the errors, rather than Fatal.

Reply:
This makes more sense to me as well, then we can have cobra worry about processing the error. I can create an issue for this.

- Relates: #16